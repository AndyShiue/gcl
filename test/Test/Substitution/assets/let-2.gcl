var a : Int
let P x = a = x

a := 3

{ P a }