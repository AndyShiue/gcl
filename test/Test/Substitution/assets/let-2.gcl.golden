P   a  [  3  /  a  ]
  * [] ===> a   =   a
    
  * [  3  /  a  ] ===> 3   =   3
    